File: app/src/main/java/x/chestnut/code/snippet/ui/recyclerView/multiItem/ItemTxtBean.java
package x.chestnut.code.snippet.ui.recyclerView.multiItem;

/**
 * <pre>
 *     author: Chestnut
 *     blog  : https://juejin.im/user/676954892408824
 *     time  : 2019/3/29 22:07
 *     desc  :
 *     thanks To:
 *     dependent on:
 *     update log:
 * </pre>
 */

public class ItemTxtBean implements IMultiType{
    public String content;
    @Override
    public int getItemType() {
        return MultiItemAdapter.ITEM_TV;
    }
}

File: app/src/main/java/x/chestnut/code/snippet/ui/recyclerView/multiItem/ItemTxtBean.kt
package x.chestnut.code.snippet.ui.recyclerView.multiItem

import x.chestnut.code.snippet.ui.recyclerView.multiItem.IMultiType.ItemState.Companion.ITEM_TV

/**
 * <pre>
 * author: Chestnut
 * blog  : https://juejin.im/user/676954892408824
 * time  : 2019/3/29 22:07
 * desc  :
 * thanks To:
 * dependent on:
 * update log:
</pre> *
 */
class ItemTxtBean : IMultiType {
    var content: String? = null
    override val itemType: Int
        get() = ITEM_TV
}